### PR TITLE
fix cov value check

### PR DESCRIPTION
**Pull request by kfwilson**
this is a little one - I realized when I went into sim validators to do docs that the cov value check wasn't quite checking what I wanted it to

### CODE REVIEW

File: src/vivarium_inputs/validation/sim.py [hunk 1]
```diff
@@ -547,7 +547,7 @@ def check_covariate_values(data: pd.DataFrame):
 
     # allow the case where lower = mean = upper = 0 b/c of things like age
     # specific fertility rate where all estimates are 0 for young age groups
-    if np.all(data.value != 0) and not np.all(lower < mean < upper):
+    if not np.all(lower == mean == upper == 0 or lower < mean < upper):
```
Comment (halfhorst) [inline src/vivarium_inputs/validation/sim.py]: so, repeated back in plain english:

either *all* lower, upper mean are zero or *all* lower < mean < upper

Comment (kfwilson) [reply src/vivarium_inputs/validation/sim.py]: Yep. Because for things like fertility rate, it'll be 0 for kids so I can't just check lower < mean < upper but before I was only doing that check if there were no 0s at all, and that's not really what I want

Comment (kfwilson) [reply src/vivarium_inputs/validation/sim.py]: wait no that's not right. In english it should be all demographic groups either have upper, mean, lower are 0 or lower < mean < upper but not that all have to be 0 or all have to have lower < mean < upper
